source/components/defining-a-component.md

```diff
@@ -74,18 +74,15 @@ file at `app/components/blog-post.js`. If your component was called
 
 ### Dynamically rendering a component
 
-The `{{component}}` helper can be used to defer the selection of a component to
-run time. The `{{my-component}}` syntax would always render the same component,
-whereas using the `{{component}}` helper allows swapping the component rendered
-on the fly. This is useful in cases where, for example, you want to interact
-with different external libraries depending on the data. Using the `{{component}}`
-helper would allow you to keep those different logic well-separated.
+The `{{component}}` helper can be used to defer the selection of a component until later.
```

Comment:
Sounds good, I'll revert that. Was trying to make this easier to understand for the non-techie just picking up Ember